src/vector.rs

```diff
@@ -126,6 +128,14 @@ impl<T: Copy, U> TypedVector2D<T, U> {
     }
 }
 
+impl<T, U> TypedVector2D<T, U>
+where T: Trig + Copy + Sub<T, Output = T> {
+    /// Returns the angle between this vector and the x axis between -PI and PI.
+    pub fn angle(&self) -> Radians<T> {
```

Comment:
can we have the name reflecting the fact X axis is compared to?

Reply:
Yep sounds good, if you have suggestions let me know, maybe 
 - `x_angle`
 - `x_axis_angle`
 - `angle_from_x_axis` (a bit long but very explicit) ?

Reply:
last one would be my preference. The method involves some math, so it's ok to have the name longer